Packs/AWS-EC2/Scripts/AwsEC2SyncAccounts/README.md

```diff
@@ -0,0 +1,46 @@
+Update an AWS - EC2 instance with a list of accounts in an AWS organization, which will allow EC2 commands to run in all of them.
```

Comment:
add here "can be used by a schedule to ..."

Reply:
done